sunnypilot/selfdrive/controls/lib/latcontrol_torque_ext_base.py

```diff
@@ -118,7 +117,7 @@ def update_calculations(self, CS, VM, desired_lateral_accel):
       desired_lateral_jerk = (np.interp(self.desired_lat_jerk_time, ModelConstants.T_IDXS,
                               self.model_v2.acceleration.y) - desired_lateral_accel) / self.desired_lat_jerk_time
       self.lookahead_lateral_jerk = get_lookahead_value(predicted_lateral_jerk[LAT_PLAN_MIN_IDX:friction_upper_idx], desired_lateral_jerk)
-      if not self.use_steering_angle or self.lookahead_lateral_jerk == 0.0:
+      if self.lookahead_lateral_jerk == 0.0:
         self.lookahead_lateral_jerk = 0.0
```

Comment:
Since we're cleaning up. This line can be removed, as we already know that `self.lookahead_lateral_jerk == 0.0`.